terraform/environments/xhibit-portal/network-infrastructure.tf

```diff
@@ -226,6 +232,27 @@ resource "aws_security_group_rule" "waf_lb-outbound-importmachine" {
   source_security_group_id = aws_security_group.importmachine.id
 }
 
+resource "aws_security_group_rule" "prtg_lb-inbound-importmachine" {
+  depends_on               = [aws_security_group.prtg_lb]
+  security_group_id        = aws_security_group.prtg_lb.id
+  type                     = "ingress"
+  description              = "allow all from bastion"
+  from_port                = 0
+  to_port                  = 0
+  protocol                 = "-1"
+  source_security_group_id = aws_security_group.importmachine.id
+}
+
+resource "aws_security_group_rule" "prtg_lb-outbound-importmachine" {
+  depends_on               = [aws_security_group.prtg_lb]
+  security_group_id        = aws_security_group.prtg_lb.id
+  type                     = "egress"
+  description              = "allow all to bastion"
```

Comment:
Can you correct these descriptions? This is allowing access to import machine from the load balancer

Reply:
description              = "allow all to prtg-lb to importmachine"